meshtastic/interfaces/ble/connection.py

```
@@ -1,10 +1,13 @@
 """BLE connection management and validation."""
 
 import logging
+import math
+import numbers
 import sys
 from threading import Event, RLock
 from typing import TYPE_CHECKING, Callable
```

Ruff (0.15.4)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)